test/appium/tests/medium/test_chats_m.py

```diff
@@ -815,6 +862,10 @@ def prepare_devices(self):
 
         self.home_2 = self.device_2.recover_access(passphrase=self.sender['passphrase'],
                                                    keycard=True, enable_notifications=True)
+
+        for home in (self.home_1, self.home_2):
```

Comment:
make it with a list comprehension just to save the line :)

Reply:
done